enhancements/windows-containers/health-management.md

```diff
@@ -224,8 +227,20 @@ For Node configuration WICD will have two separate responsibilities:
 
 #### Bootstrap command
 
-When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created
-for the instance. This functionality is currently implemented within WMCB with the `init-kubelet` command.
+When run with the `bootstrap` command, WICD will do the steps required to ensure that Node is created for the instance. 
+The `bootstrap` command will start all services that have a `priority` field value of 0, and exclusively these services.
```

Comment:
Does it mean even if we want bootstrap to start more than 1 service, all the services should have the same priority?
Can we have a range instead of a fixed number to be flexible with our design to accommodate future needs?

Reply:
In addition if the bootstrap services have an ordering requirement like containerd and kubelet, how is that being expressed?

Reply:
Good questions, we tried to address them with [this](https://github.com/openshift/enhancements/pull/1018/commits/8995d1bdd14f0f5f2fd9db616bef8a49136ac905#diff-3c7c637b883f288bf20ea8e8e521c4962adef0c81b2d149955787c8baf87273fR179-R181) and [this](https://github.com/openshift/enhancements/pull/1018/commits/8995d1bdd14f0f5f2fd9db616bef8a49136ac905#diff-3c7c637b883f288bf20ea8e8e521c4962adef0c81b2d149955787c8baf87273fR242).

Reply:
> Good questions, we tried to address them with [this](https://github.com/openshift/enhancements/pull/1018/commits/8995d1bdd14f0f5f2fd9db616bef8a49136ac905#diff-3c7c637b883f288bf20ea8e8e521c4962adef0c81b2d149955787c8baf87273fR179-R181) and [this](https://github.com/openshift/enhancements/pull/1018/commits/8995d1bdd14f0f5f2fd9db616bef8a49136ac905#diff-3c7c637b883f288bf20ea8e8e521c4962adef0c81b2d149955787c8baf87273fR242).

The link takes me to the expired page.
Are you referring to the below statement ?
`Also, when there are multiple services that share the same priority, the tied services' dependencies will be examined to
help determine order of creation.`

Reply:
Whoops. Yes that statement, as well as this: 
`The bootstrap command will start all services that have a bootstrap value of true, and exclusively these services.`